src/xg.cpp

```diff
@@ -1801,7 +1958,7 @@ vector<Edge> XG::edges_to(int64_t id) const {
     for (int64_t j = t; j < f; ) {
```

Comment:
This whole function (as well as `edges_from`) should be deleted and replaced with something along the lines of `edges_on_start` and `edges_on_end`.